Tasks/AzureTestPlanV0/testLibExecutor.ts

```diff
@@ -40,11 +41,23 @@ function getMavenExec() {
     return mvnExec;
 }
 
-function getExecOptions(): tr.IExecOptions {
+function getExecOptions(output?: { stdout: string}): tr.IExecOptions {
     var env = process.env;
-    return <tr.IExecOptions>{
+
+    var execOptions = output ? <tr.IExecOptions>{
         env: env,
-    };
+        outStream: new Writable({
+            write(chunk, encoding, callback) {
+                output.stdout += chunk.toString();
+                process.stdout.write(chunk); // Stream to console
+                callback();
+            },
```

Comment:
Should we send back any error faced during writing to console through callback function?

Reply:
added changes to send back errors